README.md

```diff
@@ -275,105 +293,114 @@ fdc3.open("ChartIQ",{
+> I made an application that can be opened by the Intent Resolver via raiseIntent but I don't see my data updating, how do I get the data when it's sent?
+>
 When you develop your application you will need to add one or more intent listeners for each intent that your application can be used with.
-`const listener = fdc3.addIntentListener('ViewInstrument', 
+`const listener = fdc3.addIntentListener('ViewInstrument',
 context => { // view instrument has been requested by another application });`
 
-### Is there a way I can find out intents in advance so that if there are no applications that accept my intent I can do something else with a user request? 
-Yes. You have two options to find intents, you can 
+> Is there a way I can find out intents in advance so that if there are no applications that accept my intent I can do something else with a user request?
+>
+Yes. You have two options to find intents, you can
 `fdc3.findIntent("ViewChart", context); //context object is optional`
 OR
 `fdc3.findIntentsByContext(context);`
 
-### I don't want to use the intent resolver each time I want to send context (data) between components, is there another way? 
-FDC3 1.1 introduces the Channels API. You can connect your components to a channel either programmatically or user-led via the Finsemble Linker. 
-Channels then allow you to share context between them. A single application (component) can only join one channel at a time e.g it cannot join the purple and the yellow channel, if the user tries to do this they will be removed from the last channel joined. You can however programmatically listen to as many channels as you like from your application. 
+> I don't want to use the intent resolver each time I want to send context (data) between components, is there another way?
+>
+FDC3 1.1 introduces the Channels API. You can connect your components to a channel either programmatically or user-led via the Finsemble Linker.
+Channels then allow you to share context between them. A single application (component) can only join one channel at a time e.g it cannot join the purple and the yellow channel, if the user tries to do this they will be removed from the last channel joined. You can however programmatically listen to as many channels as you like from your application.
 Many applications can be joined to a single-channel e.g App A, B and C can all be joined by the purple channel.
 
-### What do I need to do to enable the channels so that my end users can start using it?
+> What do I need to do to enable the channels so that my end users can start using it?
+>
 Finsemble allows you to use the Linker Channels. All you need to do is add the FDC3 preload to your component then set up a [context listener](#context-listener) and Finsemble will handle the rest. Your end-users can now use the Finsemble Linker button to change the FDC3 channel they are joined to.
```

Comment:
```suggestion
Finsemble includes user interface for selecting built-in (or system) colour channels, normally used with the Finsemble Linker API. The FInsemble FDC3 implementation, if preloaded into a component, will take over the Linker UI allowing you to use it select FDC3 system channels instead. All you need to do is add the FDC3 preload to your component then set up a [context listener](#context-listener) and Finsemble will handle the rest. Your end-users can now use the Finsemble Linker button to change the FDC3 channel they are joined to.
```

@WatsonCIQ can you confirm if they also need to enable the linker in their config or whether the preload does this for you please.